test/server-multiaddr.js

```diff
@@ -0,0 +1 @@
+export var SERVER_MULTIADDR = '';
```

Comment:
smells like bad design to me. Can better be handled with an ENV variable, no?

Reply:
@John-LittleBearLabs Should we switch to the http interface on go-libp2p instead of using the file?

Reply:
@GlenDC This was a temporary automation for running the interop test. This test will likely be removed in favor of testground.

Reply:
I understand. If you do not think there is a clearer solution possible without much work, fine for me. But I imagine setting an address can just as well be done via an env variable that you can pass with your cli commands, no? Just an idea though.

Reply:
Fair enough. It's anyway just a test file, so no big deal. Feel free to keep as-is, we already talked way more about it than necessary anyway :D